src/node_metadata.cc

```diff
@@ -127,9 +127,11 @@ Metadata::Versions::Versions() {
   cjs_module_lexer = CJS_MODULE_LEXER_VERSION;
   uvwasi = UVWASI_VERSION_STRING;
 
+#ifndef NODE_SHARED_BUILTIN_AMARO_DIST_INDEX_PATH
```

Comment:
https://github.com/nodejs/node/blob/01c88f913601bc43c149d76a70e95df77d23dc1b/src/node_metadata.h#L30 should be guarded with `NODE_SHARED_BUILTIN_AMARO_DIST_INDEX_PATH` as well?

Reply:
Will add to be consistent with Undici.

Reply:
@legendecas added